test/Feature/SingularSpec.hs

```diff
@@ -79,44 +101,68 @@ spec =
         p <- request methodPost
           "/addresses"
           [("Prefer", "return=representation"), singular]
-          [json| [ { id: 100, address: "xxx" } ] |]
-        liftIO $ simpleBody p `shouldBe` [str|{"id":100,"address":"xxx"}|]
+          [json| [ { id: 102, address: "xxx" } ] |]
+        liftIO $ simpleBody p `shouldBe` [str|{"id":102,"address":"xxx"}|]
```

Comment:
Hmm.. why are you changing the `id` value here? Makes me think that one of the previous test cases succeeded in patching when it shouldn't have.

Reply:
Separate `id` from other tests

Reply:
Oh, I see. That's better.